rust-toolchain.toml

```diff
@@ -0,0 +1,14 @@
+[toolchain]
+channel = "nightly"
```

Comment:
IMO it's more secure to be more specific here, i.e. to give a concrete toolchain, not only channel - otherwise it is a perfect place for compilation issues for different users and problems with reproducibility